### ISSUE TITLE
Broken in typescript & node: `TypeError: axios_cookiejar_support_1.default is not a function`

### ISSUE THREAD

**Issue by Bnaya** [3846masa/axios-cookiejar-support#120]
typescript 2.6.2, axios-cookiejar-support v0.2.1, node lts/boron and node lts/carbon.
this ts code:
```ts
import axiosCookieJarSupport from "axios-cookiejar-support";
axiosCookieJarSupport(Axios);
```
Is transpiled to 
```js
const axios_cookiejar_support_1 = require("axios-cookiejar-support");
axios_cookiejar_support_1.default(axios_1.default);
```

which throws 
```
axiosCookieJarSupport(Axios);
                     ^
TypeError: axios_cookiejar_support_1.default is not a function
```
See https://github.com/Microsoft/TypeScript/issues/5565

The solution would be to change the `d.ts` file and remove the default export.

**Comment by 3846masa** [comment]
Sorry.
Because of `babel-plugin-add-module-exports`, default export converted to commonjs module.exports.
I decide to remove `babel-plugin-add-module-exports`, please wait to release new version (I'll rewrite now).

**Comment by 3846masa** [comment]
@Bnaya I released as v0.3.0. Please try it.

Thanks for your report.
